packages/venia-ui/lib/components/OrderHistoryPage/resetButton.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useFormApi } from 'informed';
+import { X as ClearIcon } from 'react-feather';
+
+import Icon from '../Icon';
+import Trigger from '../Trigger';
+
+const clearIcon = <Icon src={ClearIcon} size={24} />;
+
+const ResetButton = props => {
+    const { onReset } = props;
+    const formApi = useFormApi();
+
+    const handleReset = () => {
+        formApi.reset();
+        onReset();
```

Comment:
Can we add `onReset` to `propTypes` and make it optional? Would need to add a null guard around this invocation then too.

Reply:
Added in 8c923bf.